content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,186 @@
+
+- **Updated** the Istio Workload and Istio Service dashboards to improve loading time.
+  ([Issue #22408](https://github.com/istio/istio/issues/22408))
+
+- **Added** `datasource` parameter to Grafana dashboards
+  ([Issue #22408](https://github.com/istio/istio/issues/22408))
+
+- **Added** Listener Access Logs when `ResponseFlag` from Envoy is set.
+  ([Issue #26851](https://github.com/istio/istio/issues/26851))
+
+- **Added** support for `OpenCensusAgent` formatted trace export with configurable trace context headers.
+
+- **Added** Proxy config to control Envoy native stats generation.
+  ([Issue #26546](https://github.com/istio/istio/issues/26546))
+
+- **Added** Istio Wasm Extension Grafana Dashboard.
+  ([Issue #25843](https://github.com/istio/istio/issues/25843))
+
+- **Added** gRPC streaming message count proxy Prometheus `metrics istio_request_messages_total` and `istio_response_messages_total`
+  ([PR #3048](https://github.com/istio/proxy/pull/3048))
+
+- **Added** support for properly labeling traffic in client metrics for cases when the destination is not reached or is not behind a proxy.
+  ([Issue #20538](https://github.com/istio/istio/issues/20538))
+
+- **Fixed** interpretation of `$(HOST_IP)` in Zipkin and Datadog tracer address.
+  ([Issue #27911](https://github.com/istio/istio/issues/27911))
+
+- **Removed** all Mixer-related features and functionality. This is a scheduled
+removal of a deprecated Istio services and deployments, as well as
+Mixer-focused CRDs and component and related functionality.
+  ([Issue #25333](https://github.com/istio/istio/issues/25333)),([Issue #24300](https://github.com/istio/istio/issues/24300))
+
+## Installation
+
+- **Promoted** external control plane to alpha.
+  ([Issue #11](https://github.com/istio/enhancements/issues/11))
+
+- **Updated** Kiali addon to version 1.26.
+
+- **Added** support for [installing and upgrading Istio](/docs/setup/install/helm/) using [Helm 3](https://helm.sh/docs/)
+
+- **Improved** multi-network configuration so that labeling a service with `topology.istio.io/network=network-name` can
+configure cross-network gateways without using [mesh networks](/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks).
+
+- **Improved** sidecar injection to not modify the pod `securityPolicy.fsGroup` which could conflict with existing settings and secret mounts.
+ This option is enabled automatically on Kubernetes 1.19+ and is not supported on older versions.
+  ([Issue #26882](https://github.com/istio/istio/issues/26882))
+
+- **Improved** Generated operator manifests for use with `kustomize` are available in the [manifests]({{< github_tree >}}/manifests/charts/istio-operator/files) directory.
+  ([Issue #27139](https://github.com/istio/istio/issues/27139))
+
+- **Updated** install script to bypass GitHub API Rate Limiting.
+  ([Pull Request #23469](https://github.com/istio/istio/pull/23469))
```

Comment:
This will be consistent with others that don't have issues created for them. 
```suggestion
  ([PR #23469](https://github.com/istio/istio/pull/23469))
```